crates/stand-alone-test/src/bin/make_hnsw.rs

```diff
@@ -0,0 +1,29 @@
+fn main() {
+    #[cfg(feature = "stand-alone-test")]
```

Comment:
If these binary are crates in package `stand-alone-test`, we do not need to set this cfg.

Reply:
PTAL